docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+  api:
+    build: .
+    image: ucl-assistant-api
+    env_file: .env
+    environment:
+      - REDIS_URL=redis://redis:6379?ConnectTimeout=5000&IdleTimeOutSecs=180
+    ports:
+      - "127.0.0.1:3000:3000"
+    networks:
+      - redis-net
+    depends_on:
+      - redis
+
+  redis:
+    image: redis:alpine
+    hostname: redis
+    ports:
+      - '127.0.0.1:6379:6379'
+    networks:
+      - redis-net
+
+networks:
```

Comment:
Is a separate network needed, or does the default overlay network do what's needed? Just wondering what the motivation here was.